storybook/src/admin/Tooltip.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import { Tooltip } from "@comet/admin";
+import { Add } from "@comet/admin-icons";
+import { Box, Button, Card, CardContent, Stack, Typography } from "@mui/material";
+import { forwardRef, useState } from "react";
+
+export default {
+    title: "@comet/admin/Tooltip",
+};
+
+export const IconWithTooltip = {
+    render: () => {
+        return (
+            <Tooltip title="Add something">
+                <Add />
+            </Tooltip>
+        );
+    },
+
+    name: "Icon with Tooltip",
+};
+
+export const AllTooltipTriggers = {
+    render: function Render() {
```

Comment:
I'd rather ignore the SonarCloud error for now and configure it correctly in the future.

Reply:
I agree but this prevents the PR from being merged, that's why I added this

Reply:
This will be fixed by https://github.com/vivid-planet/comet/pull/2937, so I'd revert it.

Reply:
Done in 9f5aacc097c02534fd006688477c8bf79f10721a